tests/fixtures/srl/experiment.json

```diff
@@ -0,0 +1,38 @@
+{
+  "dataset_reader":{"type":"srl"},
+  "train_data_path": "tests/fixtures/data/srl",
+  "validation_data_path": "srl-model/train/64/",
+  "model": {
+    "type": "srl",
+    "text_field_embedder": {
+      "tokens": {
+        "type": "embedding",
+        "pretrained_file": "tests/fixtures/glove.6B.100d.sample.txt.gz",
```

Comment:
You could make this a little smaller by projecting the word embeddings here.  Probably not a big deal, though, just FYI.  See the decomposable attention file for how to do that.